### ISSUE TITLE
Unnecessary use of containsTag function

### ISSUE THREAD

**Issue by jonaslagoni** [asyncapi/ts-websocket-server-template#3]
#### Reason/Context
The template currently use a `containsTag` function in the `$$channel$$` file template. I think this was used in my project, but is no longer required 🤔 

#### Description
See if it can be directly removed.

**Comment by derberg** [comment]
@jonaslagoni I bet this one can be closed, can it?

**Comment by jonaslagoni** [comment]
@derberg I mean this is a custom implementation of a websocket server using different channels etc. Dont think I will have time to finish this together with https://github.com/asyncapi/dotnet-websocket-client-template think you can delete them :)

**Comment by derberg** [comment]
@jonaslagoni you mean like delete those repos?

**Comment by jonaslagoni** [comment]
Yes 😄 Even though it is a fun project I dont see I have/will take the time to finish them within the next couple of months.

**Comment by derberg** [comment]
@jonaslagoni oh, I don't like deleting someone's work 😢 (pushing away responsibility like a pro 😄 ), can you do it if you are sure you want to do it? 😄

**Comment by jonaslagoni** [comment]
@derberg I cannot no 😄 Only organization can 😈

**Comment by derberg** [comment]
@jonaslagoni ok, I lost this battle. So I should delete those 2 repos?

**Comment by jonaslagoni** [comment]
Yes please, thanks! 🙇
